checkout/access-checkout-android/src/test/java/com/worldpay/access/checkout/views/PANLengthFilterTest.kt

```diff
@@ -113,46 +113,46 @@ class PANLengthFilterTest {
         assertEquals("", filteredString)
     }
 
-    @Test
-    fun givenAnIdentifiedCardWithMultipleValidationRulesAndMaximumCardLengthIsExceededThenShouldFilterBasedOnInput() {
-        val twentyDigitCardInput = "40000000000000000000"
-
-        val cardValidationRule1 = CardValidationRule("^41\\d{0,14}$", 16, null, null)
-        val cardValidationRule2 = CardValidationRule("^40\\d{0,18}$", null, 20, null)
-        val cardBrand = CardBrand("visa", emptyList(), null, listOf(cardValidationRule1, cardValidationRule2))
-        val validationResult = ValidationResult(partial = true, complete = false)
-        given(cardValidator.validatePAN(twentyDigitCardInput)).willReturn(Pair(validationResult, cardBrand))
-
-        // Add a '0' to existing string
-        val filteredString = panLengthFilter.filter("0", 0, 1, SpannableStringBuilder(twentyDigitCardInput), twentyDigitCardInput.length, twentyDigitCardInput.length)
-
-        // Verify that no character is added
-        assertEquals("", filteredString)
-    }
-
-    @Test
-    fun givenAnIdentifiedCardWithSubRulesAndMaximumCardLengthIsExceededForSubRuleThenShouldFilterBasedOnInput() {
-        val sixteenDigitCardInput = "4000000000000000"
-
-        val cardValidationRule1SubRule = CardValidationRule("^40\\d{0,14}$", null, 16, null)
-        val cardValidationRule1 = CardValidationRule("^4\\d{0,19}$", null, 20, null, listOf(cardValidationRule1SubRule))
-        val cardBrand = CardBrand("visa", emptyList(), null, listOf(cardValidationRule1))
-        val validationResult = ValidationResult(partial = true, complete = false)
-        given(cardValidator.validatePAN(sixteenDigitCardInput)).willReturn(Pair(validationResult, cardBrand))
-
-        // Add a '0' to existing string
-        val filteredString = panLengthFilter.filter("0", 0, 1, SpannableStringBuilder(sixteenDigitCardInput), sixteenDigitCardInput.length, sixteenDigitCardInput.length)
-
-        // Verify that no character is added
-        assertEquals("", filteredString)
-    }
+//    @Test
```

Comment:
should this still be commented out?

Reply:
These tests are no longer relevant as they test 'multiple rule sets' which is no longer applicable